plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -145,6 +145,14 @@ func defaultPredicates() sets.String {
 		// GeneralPredicates are the predicates that are enforced by all Kubernetes components
 		// (e.g. kubelet and all schedulers)
 		factory.RegisterFitPredicate("GeneralPredicates", predicates.GeneralPredicates),
+
+		// Fit is determined based on whether a pod can tolerate all of the node's taints
```

Comment:
But if that breaks some of the kubelet tests for some reason, then leave this the way you have it, and we can move this to GeneralPredicates in the next release. (Though hopefully it wouldn't break any kubelet tests...)